src/bugsnag.js

```diff
@@ -188,6 +191,11 @@
     } else {
       crumb.name = truncate(crumb.name, 32);
       breadcrumbs.push(truncateDeep(crumb, 140));
+
+      // limit breadcrumb trail length, so the payload doesn't get too large
+      if (breadcrumbs.length > breadcrumbLimit) {
+        breadcrumbs = breadcrumbs.slice(0, breadcrumbLimit);
```

Comment:
If I am reading this correctly, this logic removes the newest breadcrumb in the case that the list is too long, instead of the oldest one. If that is the case, we should probably shift crumbs onto the list or slice off the first element.